integration/fuse/src/main/java/alluxio/fuse/file/FuseFileInStream.java

```diff
@@ -47,9 +49,9 @@ public class FuseFileInStream implements FuseFileStream {
   private final FileStatus mFileStatus;
   private final AlluxioURI mURI;
   private final CloseableResource<Lock> mLockResource;
-  private final PriorityBlockingQueue<ReadTask> mOrderedTaskProvider = new PriorityBlockingQueue<>();
+  private final ConcurrentSkipListSet<ReadTask> mOrderedTaskProvider = new ConcurrentSkipListSet<>();
```

Comment:
Future improvement: Understand the overhead between PriorityBlockingQueue & ConcurrentSkipListSet
- implementation
- performance
- resource